test/module/vendor/rocksdb_test.cpp

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <rocksdb/db.h>
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+
+#include "ametsuchi/impl/rocksdb_common.hpp"
+
+namespace fs = boost::filesystem;
+using namespace iroha::ametsuchi;
+
+class RocksDBTest : public ::testing::Test {
+ public:
+  void SetUp() override {
+    db_name_ = (fs::temp_directory_path() / fs::unique_path()).string();
```

Comment:
While there is similar code in other tests, having fixed directories for each test can allow to run the tests in parallel. I suggest to replace `unique_path` with test name for example, so that it will not be random.

Reply:
Unique path also allows you to run tests in parallel. The problems will be in a case if two or more tests will be located in a single directory. If we will use direct names for each test, than we need mechanics to check uniqueness on the names, or you should check it manually each time you add new test.